css/main.css

```diff
@@ -1,3 +1,248 @@
+  color: red;
+}
 
-/* Dont' forget to link this file to your HTML in the <head> */
+h5 {
+  text-align: center;
+  color: rgb(26, 25, 24);
+}
+h2 {
+  text-align: center;
+  color: rgb(138, 25, 25);
+}
+h1 {
+  color: white;
+  font-size: 3rem;
+}
+img {
+  width: 100%;
+}
+h3 {
+  color: rgb(138, 25, 25);
+  font-size: 2rem;
+}
+h4 {
+  text-align: center;
+  font-weight: bold;
+}
+.row {
+  display: flex;
+  flex-wrap: wrap;
+  margin-right: -15px;
+  margin-left: -15px;
+}
+li a {
+  // display: inline-block;
+  //text-align: center;
+  text-decoration: none;
+  color: #5a413f;
+}
+
+ul {
+  list-style: none;
+  display: flex;
+  flex-direction: column;
+}
+.top {
+  display: flex;
+  flex-direction: column;
+  justify-items: center;
+  align-items: center;
+}
+.logo {
+  display: flex;
+  justify-content: center;
+}
+/*
+.top-navigation {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+}
+*/
+.shop-now {
+  width: 9rem;
+  background-color: #f15a29;
+  border-color: #f15a29;
+  text-shadow: none;
+  border-style: none;
+  border-radius: 4px;
+  padding: 1rem 1.5rem;
+  color: white;
+  font-size: 1.125rem;
+}
+.send-button {
+  background-color: #f15a29;
+  border-color: #f15a29;
+  text-shadow: none;
+  border-style: none;
+  color: white;
+  font-size: 1.125rem;
+  text-align: center;
+}
+
+.main {
+  height: 50vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background: url('/images/NewBirthdayCake.jpg') center no-repeat;
+}
+.last-name {
+  text-align: left;
+  color: black;
+}
+
+.submit {
+  margin: 0 auto;
+  padding: 5px 15px;
+}
+
+.section-footer {
+  margin-top: 80px;
+  background-color: rgba(0, 0, 0, 0.1);
+}
+div {
+  display: block;
+}
+h3 {
+  display: block;
+  font-size: 1.17em;
+}
+
+p {
+  display: block;
+  margin-block-start: 1em;
+  margin-block-end: 1em;
+  margin-inline-start: 0px;
+  margin-inline-end: 0px;
+}
+.form-section h3 {
+  text-align: center;
+}
+.subscribe {
+  margin-top: 20px;
+}
+input {
+  width: 100%;
+  padding: 12px 20px;
+  margin: 8px 0;
+  display: inline-block;
+  border: 1px solid #ccc;
+  border-radius: 4px;
+  box-sizing: border-box;
+}
+
+.footer-last {
+  margin-top: 60px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.box {
+  display: flex;
+  margin-left: 80px;
+}
+
+
+.contact-icons-footer .fa {
+  width: 30px;
+  margin-left: 25px;
+}
+.contact-icons-footer h3 {
+  text-align: center;
+}
+.form-container {
+  border-radius: 5px;
+  padding: 20px;
+  color: white;
+}
+@media (min-width: 768px) {
+  .about-col-6 {
+    width: 50%;
+  }
+  .top {
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+  }
+  ul {
+    list-style: none;
+    flex-direction: row;
+    justify-content: space-between;
+  }
+  .contact-icons-footer {
+    display: flex;
+    align-items: center;
+    align-content: flex-end;
+    margin-left: 240px;
```

Comment:
This breaks the page as it makes the element bigger than 100%. Please remove it.